app/controllers/api/cloud_volumes_controller.rb

```diff
@@ -49,5 +51,16 @@ def options
         super
       end
     end
+
+    private
+
+    def set_collection_klasses
+      @collection_klasses ||= {}
+      @collection_klasses[:snapshots] = CloudVolumeSnapshot
+    end
+
+    def snapshots_model_method
+      :cloud_volume_snapshots
+    end
```

Comment:
not needed, we are going to reference cloud_volume_snapshot instead of having the code jump to the wrong conclusion